src/main.go

```diff
@@ -131,10 +148,32 @@ func init() {
 	}
 }
 
+func init() {
+	// init dir /var/log/ganted
+	if _, err := os.Stat("/var/log/ganted"); os.IsNotExist(err) {
```

Comment:
Maybe put the log directory in environment variables.

Reply:
Using environment variable GANTED_LOG_DIR.